requirements/requirements.txt

```diff
@@ -0,0 +1,4 @@
+# Install requirements
+Django>=3.2
+django-manager-utils>=1.4.0
+django-regex-field>=1.3.1
```

Comment:
manager utils and regex field can be updated to their newest versions since we are changing our supported django/python versions